production/deploy/azure/terraform/environment/demo/README.md

```diff
@@ -0,0 +1,80 @@
+# Demo Buyer and Seller Terraform Configurations for Azure
+
+## Overview
+
+This directory contains examples of the buyer and seller terraform modules.
+
+The operator of a buyer or seller service pair (SellerFrontEnd + Auction and BuyerFrontEnd +
+Bidding, henceforth referred to as 'stack') will deploy the services using terraform. The
+configuration of the buyer and seller modules has many different fields, so this directory is aimed
+at serving as a guide for the operator trying to bring up a fully functioning stack. The seller
+stack is meant to communicate with a seller ad service and buyer front ends; the buyer stack is
+expected to communicate only with seller front ends.
+
+## Local Setup
+
+Review the public
+[GCP cloud support and deployment guide](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_gcp_guide.md).
```

Comment:
Can you update these instruction as well please? Then I can try to follow it so that we can be confident about it.

Reply:
I removed this section for now. Let's discuss how we want to go about the local setup.